staging/src/k8s.io/component-base/logs/api/v1/options.go

```diff
@@ -346,6 +354,9 @@ func addFlags(c *LoggingConfiguration, fs flagSet) {
 	fs.VarP(VerbosityLevelPflag(&c.Verbosity), "v", "v", "number for the log level verbosity")
 	fs.Var(VModuleConfigurationPflag(&c.VModule), "vmodule", "comma-separated list of pattern=N settings for file-filtered logging (only works for text log format)")
 
+	fs.BoolVar(&c.Options.Text.SplitStream, "log-text-split-stream", false, "[Alpha] In text format, write error messages to stderr and info messages to stdout. The default is to write a single stream to stdout. Enable the LoggingAlphaOptions feature gate to use this.")
+	fs.Var(&c.Options.Text.InfoBufferSize, "log-text-info-buffer-size", "[Alpha] In text format with split output streams, the info messages can be buffered for a while to increase performance. The default value of zero bytes disables buffering. The size can be specified as number of bytes (512), multiples of 1000 (1K), multiples of 1024 (2Ki), or powers of those (3M, 4G, 5Mi, 6Gi). Enable the LoggingAlphaOptions feature gate to use this.")
```

Comment:
From the code implementation point of view, I do not see the feature gate `LoggingAlphaOptions` restrictions on these fields `c.Options.Text.SplitStream` and `c.Options.Text.InfoBufferSize`, are these missing? Or did I miss something?

Reply:
You are right, that was missing for text options. Fixed.

I copied the code although it's the same because (theoretically) the options might progress towards beta at a different pace and/or other options might be added to the structs.